client/app/components/datagrid/cells/ChangeUserOperatorStatusColumnCell.tsx

```diff
@@ -50,12 +39,13 @@ const handleUpdateStatus = async (
   }
 };
 
-export async function ChangeUserOperatorStatusColumnCell(
-  params: Readonly<ButtonRenderCellParams>,
-) {
```

Comment:
It's been a few days but if you were curious I believe that it was this component being wrapped in an `async` function that was causing the User Operator Datagrid to rerender every time we did anything. It took me a long time to figure out 😅